modules/dxp/apps/commerce-punchout/commerce-punchout-web/src/main/java/com/liferay/commerce/punchout/web/internal/theme/template/CommercePunchOutThemeTemplateContextContributor.java

```diff
@@ -32,10 +32,24 @@ public void prepare(
 
 		contextObjects.put(
 			"commercePunchOutThemeHttpHelper",
-			_commercePunchOutThemeHttpHelper);
+			new CommercePunchOutThemeHttpHelper());
```

Comment:
Hey @MrJohn1911, create an instance field and initialize it on the `activate` method. This way we'll reuse the same instance for every call of `prepare` method instead of creating a new one always. Thx